test-tool/test_receive_copy_results_copy_status.c

```diff
@@ -26,19 +27,112 @@
 #include "scsi-lowlevel.h"
 #include "iscsi-test-cu.h"
 
+
+
```

Comment:
Is it necessary to insert blank lines here? Isn't a single blank line between #include directives and code enough?

Reply:
fixed